pkg/controller/clusterinstallation/controller.go

```diff
@@ -124,43 +123,34 @@ func (r *ReconcileClusterInstallation) Reconcile(request reconcile.Request) (rec
 		return reconcile.Result{}, err
 	}
 
-	if mattermost.Spec.DatabaseType.ExternalDatabase != "" {
```

Comment:
Don't we still need to check if we're using an external DB to avoid running the DB reconciliation?

Reply:
true, fixed